server.py

```diff
@@ -41,3 +41,27 @@ def update_product(id):
 
   """
   pass
+
+
+
+@app.route('/inventory/products/', methods= ['POST'])
+def create_products():
+    """
+
+    This method will create a storage for a new product
+
+    Args:
+
+    Returns:
```

Comment:
I think we may also return the product id for the product created?
but looks good!